core/src/main/java/org/apache/gravitino/catalog/EntityCombinedFileset.java

```diff
@@ -77,6 +77,7 @@ public String storageLocation() {
   public Map<String, String> properties() {
     return fileset.properties().entrySet().stream()
         .filter(p -> !hiddenProperties.contains(p.getKey()))
+        .filter(entry -> entry.getKey() != null && entry.getValue() != null)
```

Comment:
how could `entry.getKey()` becomes `null`?

Reply:
It is not a common case, but for Java's `Map` definition, key or value can be null, and it relies on the different `Map` implementations like HashMap, TreeMap, etc, which has different requirement.